spacemesh/v2/tx.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+
+import "google/rpc/status.proto";
+
+package spacemesh.v2;
+
+message Transaction {
+  oneof versioned {
+    TransactionV1 v1 = 1;
+  }
+}
+
+// An immutable Spacemesh transaction.
+// do not include mutable data such as tx state or result.
+message TransactionV1 {
+  bytes id = 1;
+  string principal = 2;
+  string template = 3;
+  uint32 method = 4; // this is actually limited by uint8, but no type for that.
+  Nonce nonce = 5;
+  LayerLimits limits = 6;
```

Comment:
need to double check but pretty sure this is unused, not sure we want to include this

Reply:
confirmed, layer limits for tx are not currently enabled. let's remove this for now for simplicity and to avoid confusion.